client/profile-wizard/index.js

```diff
@@ -221,18 +205,14 @@ class ProfileWizard extends Component {
 			activePlugins,
 			isJetpackConnected,
 			notes,
-			profileItems,
 			updateNote,
 			updateProfileItems,
 			connectToJetpack,
 		} = this.props;
 		recordEvent( 'storeprofiler_complete' );
 
-		const { plugins } = profileItems;
 		const shouldConnectJetpack =
-			( plugins === 'installed' || plugins === 'installed-wcs' ) &&
-			activePlugins.includes( 'jetpack' ) &&
-			! isJetpackConnected;
+			activePlugins.includes( 'jetpack' ) && ! isJetpackConnected;
```

Comment:
Now we just check if jetpack is installed and not connected. this also means we'll get users we didn't get before if they installed jetpack before doing the OBW. I've checked and this behaviour change is ok.